.github/workflows/apk-s3-distribute.yml

```diff
@@ -0,0 +1,34 @@
+name: App Distribute CI
+
+on:
+  release:
+    types: [released]
+
+jobs:
+  build_ui_components_sample_app:
+    name: Build and push to S3
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Prepare environment
+        run: |
+          git fetch --unshallow
```

Comment:
Why do we need to fetch the repository?

Reply:
We have a gradle plugin that calculates the version base on the git commits history, that is because it is needed to get the git history